sublime-package.json

```diff
@@ -105,6 +105,22 @@
               "type": "object",
               "markdownDescription": "A dictionary that \"masks\" (disables) the capabilities registered by this language server. For instance, to disable completions, use `{\"completionProvider\": true}`. To disable hover info, use `{\"hoverProvider\": true}`. Multiple providers can be disabled. To disable a subset of a provider, use a nested dictionary. For intance, to disable only the trigger characters of a language server, but not the completion functionality itself, use `{\"completionProvider\": {\"triggerCharacters\": true}}`.",
               "properties": {
+                "callHierarchyProvider": {
+                  "type": "boolean",
+                  "default": true
+                },
+                "codeActionProvider": {
+                  "type": "boolean",
+                  "default": true
+                },
+                "codeLensProvider": {
+                  "type": "boolean",
+                  "default": true
+                },
+                "colorProvider": {
+                  "type": "boolean",
+                  "default": true
+                },
```

Comment:
I suppose all (or most, if not all) capabilities should also allow an object form, even if we don't specify the exact properties in the schema.